### ISSUE TITLE
Color Palette 

### ISSUE THREAD

**Issue by Ducky6944** [abalejr/forest-focus-theme#1]
Is there a color palette available? I'm trying to replicate this theme in other places like iTerm2 but wasn't sure if you had a 0-16 color palette available so I don't have to dig through your css/json files. 

Beautiful theme by the way, thank you!

**Comment by abalejr** [comment]
Hey @typkrft sorry I'm just seeing this! Unfortunately I do not have a palette though I do plan on putting one together eventually and updating the theme a bit to use it more consistently.

This theme started out as just my workspace-specific theme in Sublime Text that I would just make surgical changes to whenever I noticed something I didn't like or thought of something I would over the course of 3ish years. I then converted it to a VS Code theme when I switched editors and for the most part that's still what stands today so the JSON could use a lot of love - there's likely plenty of redundancies and inconsistencies in the palette currently.

I have a demanding job and a toddler though, so tbh, it will probably be a while before I have time to put together a palette and/or clean up the JSON.

That said, if you have the motivation and the time and want to make a fork and submit a PR, I would be extremely grateful!
